tools/gen_stubs.zig

```diff
@@ -2,13 +2,14 @@
 //! ./gen_stubs /path/to/musl/build-all >libc.S
 //!
 //! The directory 'build-all' is expected to contain these subdirectories:
-//! arm  x86  mips  mips64  powerpc  powerpc64  riscv64  x86_64
+//! arm  x86  mips  mips64  powerpc  powerpc64  riscv32  riscv64  x86_64
```

Comment:
I really have no idea if the changes in this file are actually meaningful; I just saw code handling `riscv64` that wasn't handling `riscv32`.

Reply:
@andrewrk since you didn't leave a comment here, just want to confirm that I can mark this resolved?

Reply:
it's fine, I'll audit that file next time I do a musl update